scripts/deploymentUtils.ts

```diff
@@ -243,6 +244,7 @@ export async function deployAllContracts(
     validatorWalletCreator,
     rollupCreator,
     deployHelper,
+    espressoTEEVerifierMock,
```

Comment:
Does this constant get pulled from deployCreatorAndCreateRollup.ts?

Reply:
no it does not, I had to delete this. Thanks!

Reply:
https://github.com/EspressoSystems/nitro-contracts/pull/25/commits/1fd436a405dd77585bd39bbc7477462ba5e11b2d